test/distributed/org/apache/cassandra/distributed/test/accord/AccordCQLTestBase.java

```diff
@@ -91,6 +92,27 @@ public static void setupClass() throws IOException
         SHARED_CLUSTER.schemaChange("CREATE TYPE " + KEYSPACE + ".person (height int, age int)");
     }
 
+    @Override
+    protected void test(FailingConsumer<Cluster> fn) throws Exception
+    {
+        test("CREATE TABLE " + qualifiedAccordTableName + " (k int, c int, v int, primary key (k, c)) WITH " + transactionalMode.asCqlParam(), fn);
```

Comment:
not related to this patch, but something found... several tests call `test` but that defines `'full'` and doesn't respect the param... so overriding lets me fix those tests to also run in mixed